Riot/Modules/Room/MXKRoomViewController.m

```diff
@@ -2489,24 +2489,9 @@ - (void)updateCurrentEventIdAtTableBottom:(BOOL)acknowledge
                     if (acknowledge && self.isEventsAcknowledgementEnabled)
                     {
                         // Indicate to the homeserver that the user has read this event.
-                        
-                        // Check whether the read marker must be updated.
-                        BOOL updateReadMarker = _updateRoomReadMarker;
-                        if (updateReadMarker && roomDataSource.room.accountData.readMarkerEventId)
-                        {
-                            MXEvent *currentReadMarkerEvent = [roomDataSource.mxSession.store eventWithEventId:roomDataSource.room.accountData.readMarkerEventId inRoom:roomDataSource.roomId];
-                            if (!currentReadMarkerEvent)
-                            {
-                                currentReadMarkerEvent = [roomDataSource eventWithEventId:roomDataSource.room.accountData.readMarkerEventId];
-                            }
-                            
-                            // Update the read marker only if the current event is available, and the new event is posterior to it.
-                            updateReadMarker = (currentReadMarkerEvent && (currentReadMarkerEvent.originServerTs <= component.event.originServerTs));
-                        }
-                        
                         if (self.navigationController.viewControllers.lastObject == self)
                         {
-                            [roomDataSource.room acknowledgeEvent:component.event andUpdateReadMarker:updateReadMarker];
+                            [roomDataSource.room acknowledgeEvent:component.event andUpdateReadMarker:NO];
```

Comment:
We should probably call the new method `updateReadMarkerEventIdAtTableBottom` from this block instead of calling it by default at the end of the function

Don't you think we may provide the `component.event` to this method. If the read marker can be moved on this provided event, we will avoid to look for a second time. If you agree, don't forget to check `bubbleData.collapsed` before providing the event